content/en/docs/ambient/getting-started/enforce-auth-policies/index.md

```diff
@@ -0,0 +1,121 @@
+
+{{< text syntax=bash snip_id=deploy_l4_policy >}}
+$ kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1
+kind: AuthorizationPolicy
+metadata:
+  name: productpage-viewer
+  namespace: default
+spec:
+  selector:
+    matchLabels:
+      app: productpage
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+        principals:
+        - cluster.local/ns/default/sa/bookinfo-gateway-istio
+EOF
+{{< /text >}}
+
+If you open the Bookinfo application in your browser (`http://localhost:8080/productpage`), you will see the product page, just as before. However, if you try to access the `productpage` service from a different service account, you should see an error.
+
+Let's try accessing Bookinfo application from a `sleep` pod:
+
+{{< text syntax=bash snip_id=deploy_sleep >}}
+$ kubectl apply -f {{< github_file >}}/samples/sleep/sleep.yaml
+{{< /text >}}
+
+Since the `sleep` pod is using a different service account, it will not have access the `productpage` service:
+
+{{< text bash >}}
+$ kubectl exec deploy/sleep -- curl -s "http://productpage:9080/productpage"
+command terminated with exit code 56
+{{< /text >}}
+
+## Enforce Layer 7 authorization policy
+
+To enforce Layer 7 policies, you first need a {{< gloss "waypoint" >}}waypoint proxy{{< /gloss >}} for the namespace. This proxy will handle all Layer 7 traffic entering the namespace.
+
+{{< text syntax=bash snip_id=deploy_waypoint >}}
+$ istioctl x waypoint apply --enroll-namespace --wait
+waypoint default/waypoint applied
+namespace default labeled with "istio.io/use-waypoint: waypoint"
+{{< /text >}}
+
+You can view the waypoint proxy and make sure it has the `Programmed=True` status:
+
+{{< text bash >}}
+$ kubectl get gtw waypoint
+NAME       CLASS            ADDRESS       PROGRAMMED   AGE
+waypoint   istio-waypoint   10.96.58.95   True         42s
+{{< /text >}}
+
+Adding a [L7 authorization policy](/docs/ambient/l7-features/) will explicitly allow the `sleep` service to send `GET` requests to the `productpage` service, but perform no other operations:
+
+{{< text syntax=bash snip_id=deploy_l7_policy >}}
+$ kubectl apply -f - <<EOF
+apiVersion: security.istio.io/v1
+kind: AuthorizationPolicy
+metadata:
+  name: productpage-viewer
+  namespace: default
+spec:
+  targetRefs:
+  - kind: Service
+    group: ""
+    name: productpage
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+        principals:
+        - cluster.local/ns/default/sa/sleep
+    to:
+    - operation:
+        methods: ["GET"]
+EOF
+{{< /text >}}
+
+Note the `targetRefs` field is used to specify the target service for the authorization policy of a waypoint proxy. The rules section is similar as before, but this time we added the `to` section to specify the operation that is allowed.
```

Comment:
better links can go here once we land the auth policy doc updates in istio/istio.io#15072

Reply:
added the link.